docs/get-started/tutorials/pulsar.md

```diff
@@ -0,0 +1,207 @@
+---
+title: Ingest from Pulsar
+description: A short tutorial describing how to set up Quickwit to ingest data from Pulsar in a few minutes
+tags: [pulsar, integration]
+icon_url: /img/tutorials/pulsar.svg
+sidebar_position: 5
+---
+
+In this tutorial, we will describe how to set up Quickwit to ingest data from Pulsar in a few minutes. First, we will create an index and configure a Pulsar source. Then, we will create a Pulsar topic and load some events from the [Stackoverflow dataset](link to SO) into it. Finally, we will execute some search.
+
+## Prerequisites
+
+You will need the following to complete this tutorial:
+- A local running [Quickwit instance](/docs/get-started/installation)
+- A local running [Pulsar instance](https://pulsar.apache.org/docs/next/getting-started-standalone/). 
+
+### Quickwit setup
+
+[Download](/docs/get-started/installation) Quicwkit and start a server. Then open a new terminal to execute CLI commands with the same binary. 
+
+```bash
+./quickwit run
+```
+
+Test the cluster is running:
+
+```bash
+./quickwit index list
+```
+
+### Pulsar setup
+
+```
+wget https://archive.apache.org/dist/pulsar/pulsar-2.11.0/apache-pulsar-2.11.0-bin.tar.gz
+tar xvfz apache-pulsar-2.11.0-bin.tar.gz
+cd apache-pulsar-2.11.0
+bin/pulsar standalone
+```
```

Comment:
Maybe worth adding the one command docker run as well here as it's the most convenient method.

Otherwise users may run into the fact that they need java configured and installed among other things to get setup which can be a bit annoying for a tutorial.